2019Labs/CustomSecurityContent/setup/system_setup.yml

```diff
@@ -48,6 +48,7 @@
       - python3-jinja2
       - python3-pytest  # needed for unit tests + coverage
       - python3-PyYAML
+      - ninja  # needed for fast builds
```

Comment:
This is used under the hood by `build_product`, right?

Reply:
Yes, exactly.